install inode plugin as systemd or upstart services

jbabe opened the pull request:
- install requirements via pip
- Update readme

darkodosenovic committed:
Adding Newrelic inode monitoring playbook.

darkodosenovic committed:
Refactoring template for ubuntu 16.04.

- Needed to add absolute paths and set working directory because the
  newrelic_inode_agent executable must be run as a relative path to find
  the config file

Commit:
set role requirements

- install requirements via pip
- Update readme and Galaxy meta information

rubene commented on tasks/main.yml:
```diff
@@ -1,2 +1,58 @@
+   - name: Install inode monitor on server
+     git:
+       repo: https://github.com/Foild/newrelic-inode-plugin.git
+       dest: /usr/local/bin/newrelic-inode-plugin
```
this `/usr/local/bin/newrelic-inode-plugin` is getting use on different places can be a role deafult?

jbabe replied:
yep...added a var now.

rubene commented on tasks/main.yml:
```diff
@@ -1,2 +1,58 @@
+     args:
+       chdir: /usr/local/bin/newrelic-inode-plugin
+
+   - file:
```
some of the task have names some no, can you add a name to describe what each task is doing?

jbabe replied:
yeah - added now (no names breaks `ansible-lint` too)

rubene reviewed (commented):
💩

Commit:
resolves linting errors

- Update playbook to make ansible-lint . return a zero exit status

- Moves previous commits from a playbook into the structure for a role

- Sets some role defaults and raises errors when variables are not set

- Applies workarounds for git clone error that appeared when version was defined (version is defined so ansible-lint passes)

See Also:
- https://github.com/ansible/ansible-modules-core/issues/5504
- https://github.com/ansible/ansible/pull/19057

Commit:
use role variables in the template

- also added an ansible managed message to indicate these shouldn't be
  changed locally
- Added the bundle install back cause the script needs
  https://rubygems.org/gems/newrelic_plugin

jbabe commented:
👋

rubene commented:
🐘 🐘 🐘
